resources/views/ama_list.blade.php

```diff
@@ -0,0 +1,97 @@
+@extends('layouts/admin')
+
+
+@section('title')
+
+	AMAs list
+
+@endsection
+
+@section('content')
+		
+	
+              <div class="col-sm-12">
+                <table class="table table-bordered table-striped">
+
+                  <thead>
+                    <tr>
+                      <th>Title:</th>
+                      <th>Answred by:</th>
+                      <th>Tags:</th>
+                      <th>Actions</th>
+                    </tr>
+                  </thead>
+
+                  <tbody>
+                  	@foreach ($amas as $amas)
+                  	<tr>
+                    <td>{{$amas->title}}</td>
+                    <td>{{$amas->person}}</td>
+                    <td>{{$amas->tags}}</td>
+                    <td>
+                    
+                          <button type="button" class="btn btn-info">
+                          	View Details
+                      	  </button>
+                        
+                          <button type="button" class="btn btn-info">
+                          	Edit
+                      	  </button>
+                       
+                          <button type="button" class="btn btn-info">
+                          	Delete
+                          </button>
+                        
+                    </td>
+                	</tr>
+                    @endforeach
+                  </tbody>
+                </table>
+
+
+          <p> Filter data by: </p>
+          	<form action="">
+		        
+		        From:<input type="date" min="2018-01-01" max="2019-12-31" name="date_from"> 
```

Comment:
It's a bad practice to put the date statically like this. Imagine if you have multiple files with a date range like this - at the end of the year, you'll need to go and update all one by one. You can remove the restrictions as this is admin view and the admin should be able to see all amas from the past, no matter when they're created.